lms/djangoapps/courseware/views.py

```diff
@@ -739,6 +739,23 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
+
+    if price:
+        return "{}{}".format(currency_symbol, price)
+    else:
+        # Translators: Course Price is 0
```

Comment:
this is kind of vague. Suggest: "This refers to the cost of the course. In this case, the course costs nothing so it is available for free."